tests/unit/schemas/test_pii.py

```diff
@@ -101,10 +101,20 @@ def test_parse_birthdate(self):
         assert record.birth_date == datetime.date(1980, 1, 1)
         record = pii.PIIRecord()
         assert record.birth_date is None
+        record = pii.PIIRecord(birthdate="02-09-39")
+        assert record.birth_date == datetime.date(1939, 2, 9)
```

Comment:
nitpick; I'm wondering if we should just delete this assertion.  I believe it's the testing the same assertion on line 107, and it will fail in 14 years (the one on 107 fails in 49 years).  14 years is a long time from now, but I want to think this code will still be running then 😂 .  

Honestly, not a big deal, just thinking it doesn't add much additional value over 107.

Reply:
Yeah that's totally fair. The only reason this is in here is because it's the date of a specific edge case in the NBS exception set, but the concept is already sufficiently tested. I don't think we need this one.